sunraster/spectrogram_sequence.py

```diff
@@ -17,97 +18,32 @@
 SPECTRAL_AXIS_NAME = "spectral"
 
 
-class RasterSequence(NDCubeSequence):
+class SpectrogramSequence(NDCubeSequence, SpectrogramABC):
     """
-    Class for holding, slicing and plotting spectrogram data.
+    Class for holding, slicing and plotting a sequence of spectrogram cubes.
 
-    This class contains all the functionality of its super class with
-    some additional functionalities.
+    Spectrogram cubes can be 2D or higher.
```

Comment:
how high? can it not only go to three?

Reply:
I don't actually think ```SpectrogramCube``` or ```SpectrogramSequence``` enforce an upper limit of dimensions, just so long as the expected dimensions are somewhere it there...